README.md

```diff
@@ -1,10 +1,19 @@
 Kion CLI
 ========
-
 Kion CLI is a simple tool allowing Kion customers to generate short term access
 keys via the command line.
 
-![kion-cli usage](doc/kion-cli-usage.gif)
+![kion-cli usage](doc/kion-cli-usage.gif "kion-cli usage")
+
+Goals
+-----
+
+- Well documented usage
+- Simple, maintainable code base
+- Use the Kion public API for stability
+- Standard configuration precedence (flag > env var > config file > default value)
+- Autocompletion for Bash and ZSH
+- Wizards to gather unset but required configurations
```

Comment:
This looks to be from an old version of the README before we published to github.  Did you intend to include this change?